<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimulatorComponent</class>
 <widget class="QTabWidget" name="CSimulatorComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tb_LiveData">
   <attribute name="title">
    <string>Live data</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_LiveData">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="swift::gui::views::CNameVariantPairView" name="tvp_LiveData">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_Internals">
   <attribute name="title">
    <string>Information</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_Internals">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="swift::gui::views::CNameVariantPairView" name="tvp_Internals">
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pb_RefreshInternals">
      <property name="text">
       <string>refresh</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tb_Messages">
   <attribute name="title">
    <string>Messages</string>
   </attribute>
   <layout class="QVBoxLayout" name="vl_Messages">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="swift::gui::components::CStatusMessagesDetail" name="comp_StatusMessages">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CNameVariantPairView</class>
   <extends>QTableView</extends>
   <header>gui/views/namevariantpairview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CStatusMessagesDetail</class>
   <extends>QFrame</extends>
   <header>gui/components/statusmessagesdetail.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
